src/app/organizations/state/organizations.query.ts

```diff
@@ -36,9 +36,10 @@ export class OrganizationsQuery extends Query<OrganizationsState> {
     searchName = searchName.toLowerCase();
     if (organizations) {
       return searchName
-        ? organizations.filter(
-            organization => organization.name.toLowerCase().includes(searchName) || organization.topic.toLowerCase().includes(searchName)
-          )
+        ? organizations.filter(organization => {
+            const matchOptions: any[] = Object.values(organization).filter(objectValue => typeof objectValue === 'string');
```

Comment:
Don't feel too strongly about this, but this is going to match against the avatarurl as well, which could potentially albeit rarely lead to a slightly confusing experience

Reply:
I'd agree, I would not really expect this to match against that (or other values that are not obviously visible as text)